packages/stark-ui/src/modules/app-sidebar/services/app-sidebar.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from "@angular/core";
+import { Observable, Subject } from "rxjs";
+import { StarkAppSidebarServiceIntf } from "./app-sidebar.service.intf";
+import { StarkAppSidebarOpenEvent } from "./app-sidebar-open-event.intf";
+
+/**
+ * StarkAppSidebarService service
+ */
+@Injectable()
+export class StarkAppSidebarService implements StarkAppSidebarServiceIntf {
```

Comment:
Could you please change the name of the class to `StarkAppSidebarServiceImpl`

Reply:
Could you please add a constructor and inject `@Inject(STARK_LOGGING_SERVICE) public logger: StarkLoggingService` in it ? 
And then, do this inside: 
```typescript
public constructor(@Inject(STARK_LOGGING_SERVICE) public logger) {
    this.logger.debug(starkAppSidebarServiceName + " loaded");
}